modules/ROOT/pages/_partials/commons/common-releasenotes.adoc

```diff
@@ -38,6 +39,34 @@ endif::[]
 
 // BEGIN: Show Maintenance Releases
 
+// BEGIN -- Release 2.8.5 (Android-only) -- Mar 2021
+ifdef::param-2-8-5[]
+
+[#maint-latest]
+[#maint-2.8.5]
+== 2.8.5 -- March 2021
+Version 2.8.5 of Couchbase Lite for {param-title} delivers a fix that removes a blocking, but extraneous, Kotlin dependency.
```

Comment:
The 2.8.4 POM file incorrectly listed Kotlin as a dependency. The 2.8.5 has the corrected POM file.

Reply:
Changed in https://github.com/couchbase/docs-couchbase-lite/pull/431/commits/f0fd9ffe5a23542b3e83e95a89a2ecc98a6e4e96 @djpongh